packages/manager/modules/cda/src/translations/Messages_fr_FR.json

```diff
@@ -1,3 +1,3 @@
 {
-  "cda_title": "Ceph cluster"
+  "cda_title": "Cloud Disk Array"
```

Comment:
**question:** If content is not meant to be translated, couldn't we use it directly as a constant?